Zavala/UIKeyCommand+Zavala.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  UIKeyCommand+Zavala.swift
+//  Zavala
+//
+//  Created by Muhammad Doukmak on 3/6/21.
+//
+
+import Foundation
+import UIKit
+
+let appDelegate = UIApplication.shared.delegate as! AppDelegate
```

Comment:
Move this inside the extension or at least mark it fileprivate.

Reply:
Thanks for the feedback! Ideally I would create another object that would encapsulate this 'link' between UIKeyCommand and AppDelegate but for this simple change I guess this will do.